flag.go

```diff
@@ -805,3 +761,18 @@ func stringifySliceFlag(usage, name string, defaultVals []string) string {
 	usageWithDefault := strings.TrimSpace(fmt.Sprintf("%s%s", usage, defaultVal))
 	return fmt.Sprintf("%s\t%s", FlagNamePrefixer(name, placeholder), usageWithDefault)
 }
+
+func flagFromFileEnv(filePath, envName string) (val string, ok bool) {
```

Comment:
This puts the precedence as:

* File
* Env Var

Especially given that the altsrc (reading from configuration files) takes a lower precedence than env vars, what do you think of swapping these two?